src/settings.ts

```diff
@@ -3,7 +3,7 @@ import * as os from 'os';
 
 import { FileSystemAdapter, Pattern } from './types';
 
-const CPU_COUNT = os.cpus().length;
+const CPU_COUNT = Math.max(os.cpus().length, 1);
```

Comment:
Can you add a comment to the code above that this is a specific hack for android?

Reply:
on Android, os.cups().length equals 0